### ISSUE TITLE
Support the "Misson remove all" command send by QGroundControl

### ISSUE THREAD

**Issue by simonCor** [diux-dev/rosettadrone#32]
I tested rosetta drone with an Inspire 1 and a Phantom 4 and was really impressed what is already possible.

The one thing bothering me when using it with QGroundcontrol is, that when I try to clear a misson from the vehicle (which is done by QGroundcontrol automatically when a new planning is started) an error is returned an one has to close QGroundcontrol and reopen it to plan a new route.

I use Rosetta Drone 1.2.1 (self compiled from master) and QGroundcontrol 3.5.3.

**Comment by sjd69** [comment]
Yeah, this is fixed in the next release. If you need to overwrite the current mission in the current version just send a new mission and it'll overwrite the existing mission.
